### ISSUE TITLE
Hide disabled/deleted authy accounts

### ISSUE THREAD

**Issue by lukekarrys** [raycast/extensions#845]
# Extension – [Authy](https://github.com/raycast/extensions/tree/44eb3fe084b7525eb28214738c18f7300c796a62/extensions/authy/)
Author: @guga4ka

**Describe the feature and the current behavior/state.**

The Authy app on iOS has the ability to hide `Authy Accounts` and mark `Authenticator Accounts` for deletion in 48 hours (these are called `Apps` and `Services` in the extension respectively). Currently both of these are still listed in the output from this extension. It would be nice if this extension could hide those as well. I haven't yet looked into whether these properties are returned by the API, so I'm not sure how viable this is.

Here's a screenshot of what it looks like in the iOS app accounts screen vs the extension. 

<img src="https://user-images.githubusercontent.com/542108/153259338-35aa7057-8694-4be8-b3e3-749b06673cb3.jpeg" height="400" />

<img width="600" alt="Screen Shot 2022-02-09 at 10 49 08 AM" src="https://user-images.githubusercontent.com/542108/153260097-0ee2700c-334e-482e-8929-6dac0b9816fd.png">

**Who will benefit with this feature?**

I don't know of a way to delete `Authy Accounts` so anyone who has access to old ones that they no longer use will always see those in the output from this extension.

The hiding of `Authenticator Accounts` is less useful because they are permanently deleted in 48 hours, but for people who might be testing 2fa and making accounts this can be useful if they have many marked for deletion.

**Any Other info.**

I wanted to first see if this is a viable/wanted feature request, and if it is I would be happy to implement it. 😄

**Comment by guga4ka** [comment]
Have you tried to sync data with authy `Cmd+r`? In case it deleted from service, it also should remove from extension.

**Comment by lukekarrys** [comment]
> Have you tried to sync data with authy `Cmd+r`? In case it deleted from service, it also should remove from extension.

Yes, once the 48 hours have elapsed after marking an account for deletion a `Cmd+r` refreshes the list and removes it. Part of what I'm asking for is to have the extension hide it during that 48 hours.

What's more important to me is the `Authy Accounts` since those can't be deleted and I currently hide them in the iOS app, but they always take up the precious screen real estate as the top two options in the extension.

**Comment by lukekarrys** [comment]
I just downloaded the old deprecated Chrome App that this extension is based off and it also shows all hidden apps. It does show that `Services` contains a `marked_for_deletion` property to hide a service during the 48 hour waiting period, but as I mentioned above I'm not as concerned with this feature since it will always resolve itself in time.

Evaluating my issue from a different angle, would you be open to a preference allowing specific accounts and services to be hidden? It could use a similar UI to the `My Schedule` extension:

<img width="315" alt="Screen Shot 2022-02-09 at 1 53 48 PM" src="https://user-images.githubusercontent.com/542108/153288505-c17922dd-6dca-4bc4-92e0-5d581e483e44.png">

**Comment by guga4ka** [comment]
All of these options are a device spec, which means if you hide Authy account from one device, it still will show on another.  The same with service/authenticator accounts, if you have to delete it from one device it still will exist for 48 on another. This information couldn't get by API. 

But I can try to implement it on extension, I mean hide/show some records, but I couldn't imagine the right UX for it.
I think I can add 2 more actions, `Hide a token` and `Show hidden tokens` with a new section `Hidden` on the bottom of the list. Please contact me at slack for further ideas.
